Services/Identity/IdentityServer/SeedData.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Brock Allen & Dominick Baier. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+
+using System;
+using System.Linq;
+using System.Security.Claims;
+using HardwareHero.Services.Shared.Data;
+using HardwareHero.Services.Shared.Models.Identity;
+using HardwareHero.Services.Shared.Models.UserManagementService;
+using IdentityModel;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Serilog;
+using static Duende.IdentityServer.Models.IdentityResources;
+
+namespace IdentityServer
+{
+    public class SeedData
+    {
+        public static void EnsureSeedData(string connectionString)
+        {
```

Comment:
Why do wou have seed data and di for context, loggining and identity in one place?